src/App.js

```diff
@@ -1,30 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import BooksForm from './components/BookForm';
+import BooksList from './components/BookList';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <BooksList />
+    <BooksForm />
+  </div>
+);
```

Comment:
- [ ] Since this project uses Redux with React, we need to use the Redux approach to separate React components into presentation and container components. Putting a simple way, presentational components are not aware of Redux existence while container components are the ones connected with Redux. You can check [this link](https://redux.js.org/basics/usage-with-react#presentational-and-container-components) to read more about it. `App` component does not connect with Redux and, therefore, should be placed at `src/components` directory (directory for presentational components).